packages/eslint-plugin-next/src/utils/url.ts

```diff
@@ -1,6 +1,8 @@
 import * as path from 'path'
 import * as fs from 'fs'
 
+import { normalizeAppPath } from '../../../next/src/shared/lib/router/utils/app-paths'
```

Comment:
We can't import relatively from the `next` package like this as they are published independently of each other. We would need to copy the logic in completely